highlightjs-element.html

```diff
@@ -21,63 +24,62 @@
 
     <highlightjs-element>
       <template>
-    
+
         <link rel="import" href="/components/polymer/polymer.html">
-        <polymer-element name="proto-element">
-        <template>
-          <span>I'm <b>proto-element</b>. Check out my prototype.</span>
-        </template>
-        <script>
-          Polymer({
-          });
-        </script>
-        </polymer-element>
-    
+        <dom-module name="my-element">
+          <template>
+            <span>I'm <b>my-element</b>. Check out my prototype.</span>
+          </template>
+          <script>
+            Polymer({is: 'my-element'});
+          </script>
+        </dom-module>
+
       </template>
     </highlightjs-element>
 
-@class highlightjs-element
-@blurb Element wrapper for the highlightjs library.
-@status alpha
-@snap snap.png
+@element highlightjs-element
+@group Molecules
+@hero hero.svg
+@demo demo/index.html
 -->
-<polymer-element name="highlightjs-element" attributes="text">
 
+<dom-module id="highlightjs-element">
   <template>
-
-    <style>
-      :host {
-        display: block;
-      }
-    </style>
-
-    <link href="../highlightjs/styles/default.css" rel="stylesheet" />
-    <link href="../highlightjs/styles/atelier-forest.light.css" rel="stylesheet" />
-
-    <pre id="content"></pre>
-
+    <style include="sunburst-theme"></style>
```

Comment:
Yes it really should, but out of the box the stylesheets won't work in the main document if you're using native shadow dom. They'd need to be rewritten or we need to come up with a technique to load them dynamically. There are a couple of bug around this (noted in the code) so I didn't both trying to make it work just yet.